packages/core/src/App/Containers/Layout/header/header.jsx

```diff
@@ -11,7 +11,10 @@ const Header = ({ is_logged_in }) => {
     const { is_appstore } = React.useContext(PlatformContext);
     const { pathname } = useLocation();
     const trading_hub_routes =
-        pathname === routes.traders_hub || pathname.startsWith(routes.account) || pathname.startsWith(routes.cashier);
+        pathname === routes.traders_hub ||
+        pathname.startsWith(routes.account) ||
+        pathname.startsWith(routes.cashier) ||
+        routes.compare_cfds;
```

Comment:
```suggestion
        routes.compare_cfds;
```
shouldn't we check this with pathname?